node/src/sub_lib/utils.rs

```diff
@@ -251,6 +258,41 @@ pub struct MessageScheduler<M: Message> {
     pub delay: Duration,
 }
 
+#[cfg(target_os = "windows")]
+pub fn wsa_startup_init() {
+    let lp_vendor: *mut u8 = 1 as *mut u8;
+    let wsdata: *mut windows_sys::Win32::Networking::WinSock::WSADATA =
+        &mut windows_sys::Win32::Networking::WinSock::WSADATA {
+            wVersion: 2.2 as u16,
+            wHighVersion: 2.2 as u16,
```

Comment:
I believe the number you want here is 0x0202: a two-byte value with a 2 in the first byte and a 2 in the second byte. What I think you're getting here is 0x0002, because the floating-point value 2.2 will be truncated to 2.0 when it's converted to an integer.

Maybe the syntax you're after is `wVersion: 0x0202u16`.